src/map/utils/mobutils.cpp

```diff
@@ -189,6 +189,221 @@ namespace mobutils
+                            zoneId == ZONE_CASTLE_ZVAHL_BAILEYS ||
+                            zoneId == ZONE_CASTLE_ZVAHL_KEEP ||
+                            zoneId == ZONE_SACRIFICIAL_CHAMBER ||
+                            zoneId == ZONE_THRONE_ROOM ||
+                            zoneId == ZONE_RANGUEMONT_PASS ||
+                            zoneId == ZONE_BOSTAUNIEUX_OUBLIETTE ||
+                            zoneId == ZONE_CHAMBER_OF_ORACLES ||
+                            zoneId == ZONE_TORAIMARAI_CANAL ||
+                            zoneId == ZONE_FULL_MOON_FOUNTAIN ||
+                            zoneId == ZONE_ZERUHN_MINES ||
+                            zoneId == ZONE_KORROLOKA_TUNNEL ||
+                            zoneId == ZONE_KUFTAL_TUNNEL ||
+                            zoneId == ZONE_SEA_SERPENT_GROTTO ||
+                            zoneId == ZONE_VELUGANNON_PALACE ||
+                            zoneId == ZONE_THE_SHRINE_OF_RUAVITAU ||
+                            zoneId == ZONE_STELLAR_FULCRUM ||
+                            zoneId == ZONE_LALOFF_AMPHITHEATER ||
+                            zoneId == ZONE_THE_CELESTIAL_NEXUS ||
+                            zoneId == ZONE_LOWER_DELKFUTTS_TOWER ||
+                            zoneId == ZONE_KING_RANPERRES_TOMB ||
+                            zoneId == ZONE_DANGRUF_WADI ||
+                            zoneId == ZONE_INNER_HORUTOTO_RUINS ||
+                            zoneId == ZONE_ORDELLES_CAVES ||
+                            zoneId == ZONE_OUTER_HORUTOTO_RUINS ||
+                            zoneId == ZONE_THE_ELDIEME_NECROPOLIS ||
+                            zoneId == ZONE_GUSGEN_MINES ||
+                            zoneId == ZONE_CRAWLERS_NEST ||
+                            zoneId == ZONE_MAZE_OF_SHAKHRAMI ||
+                            zoneId == ZONE_GARLAIGE_CITADEL ||
+                            zoneId == ZONE_CLOISTER_OF_GALES ||
+                            zoneId == ZONE_CLOISTER_OF_STORMS ||
+                            zoneId == ZONE_CLOISTER_OF_FROST ||
+                            zoneId == ZONE_FEIYIN ||
+                            zoneId == ZONE_IFRITS_CAULDRON ||
+                            zoneId == ZONE_QUBIA_ARENA ||
+                            zoneId == ZONE_CLOISTER_OF_FLAMES ||
+                            zoneId == ZONE_QUICKSAND_CAVES ||
+                            zoneId == ZONE_CLOISTER_OF_TREMORS ||
+                            zoneId == ZONE_CLOISTER_OF_TIDES ||
+                            zoneId == ZONE_GUSTAV_TUNNEL ||
+                            zoneId == ZONE_LABYRINTH_OF_ONZOZO ||
+                            zoneId == ZONE_SHIP_BOUND_FOR_SELBINA ||
+                            zoneId == ZONE_SHIP_BOUND_FOR_MHAURA ||
+                            zoneId == ZONE_SHIP_BOUND_FOR_SELBINA_PIRATES ||
+                            zoneId == ZONE_SHIP_BOUND_FOR_MHAURA_PIRATES ||
+                            zoneId == ZONE_THRONE_ROOM_V))
```

Comment:
Throne Room V?

Reply:
```
156	Throne Room [S]
165	Throne Room
229	Throne Room [V]
```
`[V]` is Voracious Resurgence battlefield area I think?

Reply:
Fat fingered the copy. I’ll update when I get home.

Reply:
Fixed